nix/shell.nix

```diff
@@ -14,7 +14,7 @@ cabalProject:
     cabal-fmt.extraOptions = "--no-tabular";
     nixpkgs-fmt.enable = true;
     shellcheck.enable = true;
-    fourmolu.enable = true;
+    fourmolu.enable = false;
```

Comment:
@zeme-wana, it looks like `fourmolu` fails to parse the C preprocessor directives that I added to the Haskell source. Is there any way to fix this by alterning the `fourmolu` configuration?

Reply:
While in theory `fourmolu` [supports](https://github.com/tweag/ormolu/blob/master/DESIGN.md#cpp) CPP directives, in practice it doesn't; this fails to parse:
```
fourmolu -o -XCPP --mode inplace marlowe-plutus/src/Language/Marlowe/Plutus/Semantics.hs 
```
Upgrading fourmolu to the latest version doesn't help either.
I added `{- FOURMOLU_DISABLE -}` accordingly.
This has to be fixed upstream unfortunately